tools/osbuilder/rootfs-builder/init_trusted_storage.sh

```diff
@@ -0,0 +1,51 @@
+#!/bin/bash
+#
+# Copyright (c) 2022 Intel Corporation
+#
+# SPDX-License-Identifier: Apache-2.0
+#
+
+set -o errexit
+set -o nounset
+set -o pipefail
+
+[ -n "${BASH_VERSION:-}" ] && set -o errtrace
+[ -n "${DEBUG:-}" ] && set -o xtrace
+
+device_num=${1:-}
+if [ -z "$device_num" ]; then
+        echo "ERROR: invalid arguments" >&2
+        exit 1
+fi
+
+enable_data_integrity="true"
+if [ -n "${2-}" ]; then
+        enable_data_integrity="$2"
+fi
+
+device_name=$(cat /sys/dev/block/${device_num}/uevent|grep DEVNAME|cut -d "=" -f2)
+device_path="/dev/$device_name"
+if [[ -n "$device_name" && -b "$device_path" ]]; then
+	dd if=/dev/urandom of=/run/cc_storage.key bs=1 count=4096
```

Comment:
Just curious if setting up a random key each time is intended to be the long-term solution, or just a temporary kludge until we get actual keys from the KBS? If the latter, probably need to add another option that distinguishes between ephemeral storage (where you can generate the key that way) and persistent storage (where you would need to get the key from the agent).

Reply:
Yes, current is for init empheral storage, when we start support persistent storage, we will need get key from the KBS.